.github/workflows/fixxxer.yaml

```diff
@@ -0,0 +1,75 @@
+name: PR Fixxxer
+
+on:
+  issue_comment:
+    types: [created]
+
+permissions:
+  contents: write
+
+jobs:
+  pr_commented:
+    name: Run PR Fixxxer
+    # TODO: support leading and trailing whitespace too
+    if: ${{ github.event.issue.pull_request && github.event.comment.body == '/fixxx' }}
```

Comment:
```suggestion
    if: ${{ github.event.issue.pull_request && startsWith(github.event.comment.body, '/fixxx') }}
```

Reply:
I'm not sure I like this. For example it does not help with the common use case when you copy/paste the command from an earlier comment, because GH prepends a newline. OTOH it will also act on `/fixxxwithoutspace` which is not great... 🤔 
I'd rather do a bit more elaborate parsing later...